.github/workflows/e2e-testing.yml

```diff
@@ -58,7 +53,7 @@ jobs:
           docker compose logs provisioning-agent > provisioning_agent_logs.txt 2>&1 || true
           cat provisioning_agent_logs.txt
           # Fail only if a single line contains both "azure-init" and "failed"/"failure" (case-insensitive).
-          if grep -i 'azure-init' provisioning_agent_logs.txt | grep -Ei 'failed|failure' >/dev/null; then
+          if grep -i 'azure-init' provisioning_agent_logs.txt | grep -Ei 'failed|failure' | grep -v 'sshd -G' >/dev/null; then
```

Comment:
be more specific on the error message being ignored.  fix it (separately) if it's the wrong level (or spurious error)